catkin_ws/src/sensors/launch/display.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+    <node name="display_serial_server" pkg="rosserial_python" type="serial_node.py" respawn="true">
+        <param name="port" value="/dev/ttyACM0"/>
```

Comment:
This port will need to change at some point with udev rules but this is fine for now